docker/test-server-mysql.yml

```diff
@@ -0,0 +1,54 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Setup for a test server with MySQL
+# ----------------------------------------------------------------------------------------------------------------------
+# It is designed to take in a lot of environment variables to take in all the configuration of the test server.
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+    artemis-app:
+        extends:
+            file: ./artemis.yml
+            service: artemis-app
+        image: ghcr.io/ls1intum/artemis:${ARTEMIS_DOCKER_TAG:-latest}
```

Comment:
Is the environment variable ``${ARTEMIS_DOCKER_TAG:-latest}`` in sync with [this line inside the ansible PR](https://github.com/ls1intum/artemis-ansible-collection/pull/34/files#diff-eae1f3b0a6e12a43a32d479475595d15fce91ae52f2c588d141c519888e0c618R33)?

Reply:
The line in the ansible collection was not correctly updated. Fixed now.